src/Logger.cc

```diff
@@ -237,8 +237,8 @@ LogSeverity Logger::logLevel( Logger *logger )
 
     if ( logger )
         return logger->logLevel();
-    else
-        return LogSeverityVerbose;
+
```

Comment:
Why?

This was a conscious decision: The control flow is clear. If there is a logger ( the absolutely normal case), that logger's log level is used, otherwise a fallback value.

There is no way that none of those branches are executed; if it were, and the function would not return a value, the compiler would complain. But with this change, it appears as if the fallback were a normal case, not the exception.